project.clj

```diff
@@ -1,7 +1,8 @@
 (defproject circleci/mongofinil "0.2.20"
   :description "A library for Mongoid-like models"
   :dependencies [[congomongo "1.1.0"]
```

Comment:
~ If we're bumping dependencies anyway... Can you make this `2.2.1`?

Reply:
I was only doing non-core dependencies. Is 1.10 -> 2.2.1 a safe bump? Should it be done in a separate PR?

Reply:
We're running `congomongo "2.2.1"` in our production services, so I assume it's safe enough.